src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval.java

```diff
@@ -756,14 +757,24 @@ private synchronized void removePendingScript(String hash) {
         return pendingSignatures;
     }
 
+    // Added to account for new findbugs annotations in ExtensionList#get (PCT scenario)
```

Comment:
For these plugins, besides the current scenario, we are also doing full builds against both baseline and target.